### ISSUE TITLE
Import error

### ISSUE THREAD

**Issue by hadhoryth** [laclouis5/globox#6]
Python 3.8.12

Hi @laclouis5, the library looks nice, but when I'm trying to use it have an error.

```
>>> import globox
 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/conda/lib/python3.8/site-packages/globox/__init__.py", line 2, in <module>
    from .annotation import Annotation
  File "/opt/conda/lib/python3.8/site-packages/globox/annotation.py", line 11, in <module>
    class Annotation: 
  File "/opt/conda/lib/python3.8/site-packages/globox/annotation.py", line 19, in Annotation
    image_size: Optional[tuple[int, int]] = None, 
TypeError: 'type' object is not subscriptable```

**Comment by laclouis5** [comment]
Hi, I found incompatibilities with Python 3.8 that I just fixed in version 1.1.1. You can either download this new version from pip (`pip install -U globox`) or pull and checkout the `v1.1.1` GitHub release.

**Comment by hadhoryth** [comment]
Thanks for so fast update! Working will try to use it now.
